truss/tests/test_truss_handle.py

```diff
@@ -654,11 +671,16 @@ def test_container_stuck_in_created(container_state_mock):
 
 @pytest.mark.integration
 @pytest.mark.parametrize(
-    "binary",
-    [(True), (False)],
+    "binary, python_version",
+    [
+        (binary, python_version)
+        for binary in [True, False]
+        for python_version in ["3.8", "3.9"]
+    ],
 )
-def test_control_truss_local_update_flow(binary, custom_model_control):
+def test_control_truss_local_update_flow(binary, python_version, custom_model_control):
     th = TrussHandle(custom_model_control)
+    th.update_python_version(python_version)
```

Comment:
I'm not super clear on what this is doing.  What is it updating based on this python_version?

Reply:
This updated python version in truss_config, which in turn governs the python version on the docker image. Ultimately, this governs what python version users model code runs with.

Reply:
Gotcha. Does this only validate tests run in docker?

Reply:
Yes, these run with docker. What do you mean by `only validate`? These run the whole truss patch flow using the docker image built for truss which those versions.

Reply:
> whole truss patch flow 

I see. Makes sense if this is the only to test that flow. Ideally, we want to run all tests in both environments, but agreed this is a stop gap for now.